tensorboard/plugins/custom_scalar/custom_scalars_plugin.py

```diff
@@ -286,15 +285,11 @@ def layout_impl(self):
     title_to_category = {}
 
     merged_layout = None
-    runs = list(self._multiplexer.Runs())
+    runs = list(self._multiplexer.PluginRunToTagToContent(metadata.PLUGIN_NAME))
     runs.sort()
     for run in runs:
-      try:
-        tensor_events = self._multiplexer.Tensors(
-            run, metadata.CONFIG_SUMMARY_TAG)
-      except KeyError:
-        # This run lacks a layout.
-        continue
+      tensor_events = self._multiplexer.Tensors(
```

Comment:
Is this loop logic is correct?

Reply:
I think so. The runs retrieved from `PluginRunToTagToContent` are the ones relevant to the plugin.

Reply:
Ah yes, I see. GitHub and Python didn't make it clear to me that the lines that follow were part of the same scope.

Reply:
Ah, I see. Yeah, the block indentation was hidden from the conversations view. As an unrelated aside, I have several quips about GitHub's code review interface.

Reply:
Nothing will ever compare to critique.